Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM docker.platform.cloud.coe.ic.gov/centos7
+FROM $REGISTRY/centos7
```

Comment:
An ARG statement will be needed prior to the FROM statement to allow this to build. I'd parameterize the whole thing, not just the registry. Ref: https://github.com/ngageoint/scale/blob/a99f2e5f3792e2f6cc9c1e129596c7e48c3beedd/Dockerfile#L1